web-components/src/components/cards/StepCard.tsx

```diff
@@ -221,31 +174,45 @@ function StepCard({
             )}
           </div>
         </div>
-        <div className={styles.cardBottom}>
-          <Title
-            variant="h6"
-            className={cx(styles.step, step.isActive && styles.activeColor)}
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'space-evenly',
+            pt: 3.5,
+            pb: [3.5, 8],
+            px: [4, 6],
+            gap: [2, 3],
+          }}
+        >
+          <Label
+            size="sm"
+            mobileSize="sm"
+            color={step.isActive ? 'secondary.main' : 'info.dark'}
           >
             step {step.stepNumber}
-          </Title>
+          </Label>
           <Title
             variant="h4"
-            className={cx(
-              styles.stepTitle,
-              step.isActive && styles.activeTitle,
-            )}
+            mobileVariant="h6"
+            color={step.isActive ? 'primary.contrastText' : 'info.dark'}
           >
             {step.title}
           </Title>
-          <Description className={styles.stepDescription}>
+          <Body size="lg" align="center" color="info.dark">
```

Comment:
Note: the way these components function, you can pass values through `sx` or as prop values. There's a gotcha though: the prop values won't work for `styled` properties.... so if you set `marginTop` on the base element, you can't pass `mt={3}` as  a prop (but the `sx` version would work).